azkaban-hadoop-security-plugin/build.gradle

```diff
@@ -1,12 +1,20 @@
 apply plugin: 'distribution'
 
+configurations {
+    testCompile.extendsFrom compileOnly
+}
+
 dependencies {
     compile project(":azkaban-common")
 
     compileOnly deps.hadoopCommon
     compileOnly deps.hadoopMRClientCommon
     compileOnly deps.hadoopMRClientCore
     compileOnly deps.hiveMetastore
+    compileOnly(deps.hiveExec) {
+        exclude group: 'org.pentaho', module: 'pentaho-aggdesigner-algorithm'
```

Comment:
Main cause: They are not present in maven central or jcenter.
Why we don't care? These modules are not required to compile the code or run the test cases.